bin/check_formatting.dart

```diff
@@ -0,0 +1,19 @@
+import 'dart:async';
+import 'utils.dart';
+
+Future<Null> main() async {
+  final CommonUtils utils = new CommonUtils();
+
+  utils.fetchConfiglet().then((int exitCode) async {
+    if (exitCode == 0) {
+      print('Checking config.json formatting...');
+      await utils.runCmd('bin/configlet', ['lint', '.']);
```

Comment:
This should be updated to meet this command:
`configlet lint --track-id dart .` To avoid false positives about duplicate UUIDs